### PR TITLE
Fixes bug with integration app react-core import

### PR DESCRIPTION
**Pull request by dlabaj**
Fixes bug with integration app @patternfly/react-core in package.json

**What**:

**Additional issues**:

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Label/Label.tsx [hunk 1]
```diff
@@ -11,13 +11,22 @@ export interface LabelProps extends React.HTMLProps<HTMLSpanElement> {
...
 export const Label: React.FunctionComponent<LabelProps> = ({
   children,
-  className = '',
-  isCompact = false,
+  className,
+  isCompact,
   ...props
-}) => (
+}) => {
+  console.log(`isCompact = ${isCompact}`)
```
Comment (rebeccaalpert) [inline packages/patternfly-4/react-core/src/components/Label/Label.tsx]: Is there supposed to be a console.log?

File: packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -13,8 +13,8 @@ export interface AvatarProps
 }
 
 export const Avatar: React.SFC<AvatarProps> = ({
-  className = '',
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx]: These empty string defaults should be added back!

File: packages/patternfly-4/react-core/src/components/Label/Label.tsx (hunk 1 above)
Comment (dlabaj) [reply packages/patternfly-4/react-core/src/components/Label/Label.tsx]: Nope thanks.

File: packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx (hunk 2 above)
Comment (dlabaj) [reply packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx]: Thanks removed the default props like we discussed and added these back.

**Comment by dlabaj** [comment]
Closing pull request the fix will be add here https://github.com/patternfly/patternfly-react/pull/1971

**Pull request closed without merging**